components/strava/actions/get-stats/get-stats.mjs

```diff
@@ -0,0 +1,23 @@
+import strava from "../../strava.app.mjs";
+
+export default {
+  name: "Get Stats",
+  description: "Returns the activity stats of an athlete. Only includes data from activities set to Everyone visibilty. [See the docs](https://developers.strava.com/docs/reference/)",
+  key: "strava-get-stats",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    strava,
+  },
+  async run({ $ }) {
+    const athlete = await this.strava.getAuthenticatedAthlete({
+      $,
+    });
+    const resp = await this.strava.getStats({
+      $,
+      athlete,
+    });
```

Comment:
```suggestion
    const { id: athleteId } = await this.strava.getAuthenticatedAthlete({
      $,
    });
    const resp = await this.strava.getStats({
      $,
      athleteId,
    });
```
then you can modify the function `getStats` passing `athleteId` instead of `athlete`

Reply:
done